.ci_support/linux_python2.7.yaml

```diff
@@ -20,10 +22,3 @@ python:
 - '2.7'
 qt:
 - '5.6'
```

Comment:
This and the rest of the pins in this repo need to be updated to 5.9. However, I don't know how to do it.